Composer/packages/lib/shared/src/viewUtils.ts

```diff
@@ -183,19 +183,26 @@ export function getDialogGroupByType(type) {
 
 const truncateSDKType = ($kind) => (typeof $kind === 'string' ? $kind.replace('Microsoft.', '') : '');
 
+const getSchemaTitle = (sdkschema: JSONSchema7, data) => {
+  const rawTitleValue = sdkschema.title as string | ((d) => string) | undefined;
```

Comment:
did you mean to use || instead of |

Reply:
They are type declarations, I'll clarify the definition.